### PR TITLE
Feature/admin

### PR DESCRIPTION
**Pull request by tijmenbaarda**
* Fetch English names of languages
* Handle continuation annotations
* Handle term groups
* Use lidiaId for relations between annotations
* Improve admin interface

### CODE REVIEW

File: lidiabrowser/sync/populate.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+from django.db import transaction
+
+import yaml
+import logging
+
+import sync.models as syncmodels
+from sync.zoteroutils import get_attachment_url, get_attachment_id_from_url
+from lidia.models import ArticleTerm, BaseAnnotation, Category, ContinuationAnnotation, LidiaTerm, Publication as LidiaPublication, TermGroup
+from lidia.models import Annotation as LidiaAnnotation
+from lidia.models import Language  #, ArticleTerm, LidiaTerm, Category
```
Comment (ar-jan) [inline lidiabrowser/sync/populate.py:11]: `Publication as LidiaPublication` can go now that you use `syncmodels`, also old placeholder comment `#, ArticleTerm, LidiaTerm, Category`

**Review by ar-jan — APPROVED** [review]
Looks good

File: lidiabrowser/sync/populate.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    annotations = BaseAnnotation.objects.order_by(
+        "parent_attachment", "sort_index"
+    )
+    current_parent = None
+    current_first_annotation = None
+    to_be_deleted = []
+    for annotation in annotations:
+        if hasattr(annotation, "annotation"):
+            # Object is of Annotation model (first annotation)
+            current_first_annotation = annotation.annotation  # noqa
```
Comment (ar-jan) [inline lidiabrowser/sync/populate.py:30]: Do we need to change something with the linter config? Or why is it needed?

Comment (tijmenbaarda) [reply lidiabrowser/sync/populate.py:30]: Sorry, this should have been `# type: ignore`. I am using Pyright and it does not recognize `hasattr`.

**Pull request merged**